src/cloud-api-adaptor/pkg/agent/config.go

```diff
@@ -0,0 +1,50 @@
+package agent
```

Comment:
should we maybe rename the package to aa or attestation-agent?

Reply:
Yeah, I agree.